js/main.js

```diff
@@ -1,9 +1,85 @@
 var $searchBar = document.querySelector('.search-bar');
+var $locationForm = document.querySelector('.location-form');
 var $searchSubmitButton = document.querySelector('.submit-search');
-function saveLocationToData(event) {
-  if ($searchBar.value !== '') {
-    data.locations.push({ location: $searchBar.value, entryId: data.nextEntryId });
-    data.nextEntryId++;
+var $weatherInformationChoices = document.querySelector('.weather-information-choices');
+var $weatherChoicesQuestion = document.querySelector('.weather-information-choices-question');
+var $locationAsker = document.querySelector('.location-asker');
+var $weatherChoicesList = document.querySelector('.list-of-weather-choices');
+var weatherOptions = ['Main', 'Temperature', 'High', 'Low', 'Wind', 'Humidity', 'SunsetSunrise'];
+var locationWeatherInformation = {};
```

Comment:
These would probably be good candidates to store in a data model you create in `data.js`. Keeping all your relevant data specific to the user in one place will make your code able to grow more seamlessly